Wikibase/Sniffs/Commenting/DisallowedDocTagsSniff.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * Custom sniff that reports deprecated PHPDoc tags (e.g. known misspellings) and replaces them with
+ * their preferred counterparts.
+ *
+ * @license GPL-2.0+
+ * @author Thiemo Mättig
+ */
+class Wikibase_Sniffs_Commenting_DisallowedDocTagsSniff implements PHP_CodeSniffer_Sniff {
+
+	/**
+	 * @var array Set a value to null if a tag is forbidden, but no automatic fix should be made.
```

Comment:
Again, I'm not really sure what your question is. I made the list you can see here in the code based on tons of experience with this stuff, based on actual statistics I made with all our code bases (including core), and based on what PHPDoc.org [as well as Doxygen](https://www.stack.nl/~dimitri/doxygen/manual/commands.html) describe. Bad tags left, good tags right.

As said, I will document this later.